src/fromager/wheels.py

```diff
@@ -200,13 +198,33 @@ def add_extra_metadata_to_wheels(
         for req_file in req_files:
             shutil.copy(req_file, dist_info_dir / f"fromager-{req_file.name}")
 
-        cmd = ["wheel", "pack", str(dist_info_dir.parent), "-d", str(wheel_file.parent)]
+        build_tag_from_settings = ctx.settings.build_tag(req.name, version)
+        build_tag = build_tag_from_settings if build_tag_from_settings else (0, "")
+
+        cmd = [
+            "wheel",
+            "pack",
+            str(dist_info_dir.parent),
+            "-d",
```

Comment:
What does `-d` do? Is there a more verbose version of that flag name?

Reply:
It is the destination dir: https://wheel.readthedocs.io/en/stable/reference/wheel_pack.html#cmdoption-d

Should I use the more verbose name?

Reply:
Yes, please. I like long names in places like this because it means I spend less time looking at docs for commands I'm not familiar with.